flytekit/clis/sdk_in_container/pyflyte.py

```diff
@@ -20,16 +20,6 @@
 
 
 @click.group("pyflyte", invoke_without_command=True)
-@click.option(
```

Comment:
so this means that if you have `pyflyte -p proj -d domain register workflows` it'll have to be changed to `pyflyte register -p proj -d domain workflows` right?

I know I said it was okay to take out project from serialize but I guess I didn't realize what that entailed.  if we're going to break compatibility, should we get rid of pyflyte register entirely?  Maybe let's discuss with ketan?  I'm okay taking  it out completely, leaving pyflyte serialize to be the one and only command you need.

Reply:
@kumare3 any thoughts on getting rid of pyflyte register altogether?

@wild-endeavor there's also the `pyflyte lp` command group too

Reply:
I am all for simplifying this story, but what about existing users and their make files - let's say at lyft

Reply:
I thought backwards compatibility wasn't an explicit concern?

If we want to go that route then we make PROJECT, DOMAIN, VERSION optional params in the pyflyte command group and every subcommand has to check that they're secretly filled in. I see it as simpler to use click features to require the args at the most specific level where they are used